### PR TITLE
Create home page

### PR DESCRIPTION
**Pull request by QianyingLuo**

### CODE REVIEW

File: index commented.txt
Comment (devs-toni) [inline index commented.txt:1]: This has to be removed!

File: src/controller/index.py [hunk 1]
```diff
@@ -1,12 +1,40 @@
...
 
 router = APIRouter()
 
-@router.get("/")
-async def index():
-    navbar_path = "../assets/templates/base.html"
-    path = get_file_path(navbar_path)
-    return FileResponse(path)
\ No newline at end of file
+class ExtendedEnvironment(Environment):
```
Comment (devs-toni) [inline src/controller/index.py:10]: Please take a look, what is this used for?

Comment (QianyingLuo) [reply src/controller/index.py:10]: In the constructor (__init__), the int function is added to the global variables of the Jinja2 environment. This allows the int function to be used directly in Jinja2 templates in index.html without the need to explicitly import it into each template.

Comment (devs-toni) [reply src/controller/index.py:10]: Ok

**Pull request merged**